pkg/services/sqlstore/org_users.go

```diff
@@ -20,7 +20,16 @@ func init() {
 func AddOrgUser(cmd *m.AddOrgUserCommand) error {
 	return inTransaction(func(sess *DBSession) error {
 		// check if user exists
-		if res, err := sess.Query("SELECT 1 from org_user WHERE org_id=? and user_id=?", cmd.OrgId, cmd.UserId); err != nil {
+		user := new(m.User)
```

Comment:
When creating pointers to structs the preferred way is to write `user := &m.User{}` instead of using `new()`. `new()` should only be used when `&{}` is not possible. Ex when creating pointers to ints

Reply:
Good catch!